docs/components/modeler/web-modeler/design-your-process.md

```diff
@@ -0,0 +1,19 @@
+---
+id: design-your-process
+title: Design your process
+description: "Design mode is tailored to business users and allows strategic modeling"
+---
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+In the **Design** mode view, business users have access to a different workspace of Web Modeler with a reduced properties panel. Only comments are shown, which provides a decluttered user interface.
+
+All the technicalities, such as triggers to deploy the diagram or start the instance, are hidden. Linting is disabled, and problem annotations are discarded.
+
+The comments panel is collapsed when switching to the design mode, and if a user expands it, the state is persisted even when switching to another diagram.
+
+As a business user, you can **link decision models** and **process models** via [call activities](/components/modeler/bpmn/call-activities/call-activities.md), and you can still be a **project owner**, even if you don't execute implementation.
```

Comment:
@christinaausley would "Creating readable process models" be a good page to link to "process models"? Otherwise, do you mind suggesting a good content for it? Thanks! :)

Reply:
Yes! That will work 👍